addons/mail/static/src/components/discuss_sidebar_mailbox/discuss_sidebar_mailbox.xml

```diff
@@ -2,20 +2,20 @@
 <templates xml:space="preserve">
     <t t-name="mail.DiscussSidebarMailbox" owl="1">
         <t t-if="mailbox">
-            <div class="o_DiscussSidebarMailbox"
+            <div class="o_DiscussSidebarMailbox d-flex align-items-center py-1 px-0 btn border-0 font-weight-normal text-reset"
                 t-att-class="{
-                    'o-active': mailbox === messaging.discuss.thread,
+                    'o-active bg-200': mailbox === messaging.discuss.thread,
                     'o-starred-box': mailbox === messaging.starred,
                 }" t-attf-class="{{ className }}" t-on-click="mailbox.onClick" t-att-data-thread-local-id="mailbox.localId" t-att-data-thread-name="mailbox.displayName"
                 t-ref="root"
             >
-                <ThreadIcon className="'o_DiscussSidebarMailbox_item'" threadLocalId="mailbox.localId"/>
-                <div class="o_DiscussSidebarMailbox_item o_DiscussSidebarMailbox_name">
+                <ThreadIcon className="'o_DiscussSidebarMailbox_item ml-4 mr-2'" threadLocalId="mailbox.localId"/>
```

Comment:
We've talked about this topic before, and I think you're right. However, I'm not so sure about this point, because if we are adapting this right now to the syntax of BS5, we should apply it to the whole of the changes done so far to our current components. Actually, this would imply changing the whole BS4 syntax currently used in Odoo to BS5.

What do you think about this @stefanorigano ?

Reply:
There are two ongoing process at the moment; the migration to BS5 (handled by mobile team) and  our SCSS ref.
Rather than waiting for the migration to be completed, or re-do the SCSS twice (once for BS4 and yet again for BS5), we decided to introduce BS5 classes to get a "BS5 ready" result during our ref.

Mobile will take care of the real migration but meanwhile we're refactoring the code we try to ease their job.

Reply:
Alright, it looks clear to me 👍 

I'll adapt as far as possible my current BS4 classes to the BS5 syntax.